vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs

```diff
@@ -36,15 +36,14 @@ type internal FsiCommandFilter(serviceProvider: System.IServiceProvider) =
 
     interface IOleCommandTarget with
         member x.Exec (pguidCmdGroup, nCmdId, nCmdexecopt, pvaIn, pvaOut) =
-            if pguidCmdGroup = VSConstants.VsStd11 then
-                if nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
-                elif nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteLineInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteLine null null
+            if pguidCmdGroup = VSConstants.VsStd11 && nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
+                do Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
```

Comment:
@JoshVarty welcome to F# programming.

Please loose the do, in general match the style of the surrounding code is the watchword.  In this case the do is unnecessary. 

Kevin